src/main/scala/uk/gov/nationalarchives/dp/client/EntityClient.scala

```diff
@@ -108,6 +137,20 @@ object EntityClient {
       } yield entities
     }
 
+    override def entityEventActions(
+        entity: Entity,
+        secretName: String,
+        startEntry: Int = 0,
+        maxEntries: Int = 1000
+    ): F[Seq[EventAction]] = {
+      val queryParams = Map("max" -> maxEntries, "start" -> 0)
```

Comment:
Does this need to be `"start" -> startEntry` If it's possible that the calling client will want to get event actions starting from somewhere other than zero then that makes sense. If they're always going to be getting event actions from the start, do they need the startEntry argument at all?

Reply:
Ah, yes, it should be `"start" -> startEntry`; good catch.